src/main/java/skibidi/DeadlineCommand.java

```diff
@@ -0,0 +1,51 @@
+package skibidi;
+
+import java.io.IOException;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.LocalDate;
+
+import java.util.Arrays;
+
+public class DeadlineCommand extends Command {
+    public static final String COMMAND_WORD = "deadline";
+    private final String description;
+    private final String by;
+    private final MilitaryTime time;
+
+    public DeadlineCommand(String[] input) throws SkibidiException, DateTimeParseException {
```

Comment:
Please add the header comment for this public method. I noticed the same issue for other public methods as well.